src/vse_sync_pp/analyzers/ppsdpll.py

```diff
@@ -0,0 +1,34 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+
+"""Analyze ppsdpll log messages"""
+
+from .analyzer import Analyzer
+
+### TODO
+#prtc_b = 40
+#prtc_a = 100
+
+class PhaseOffsetTimeErrorAnalyzer(Analyzer):
+    """Analyze DPLL phase offset time error"""
+    id_ = 'ppsdpll/phase-offset-time-error'
+    parser = 'dpll/phase-offset'
+    cols = ('phaseoffset',)
+    def prepare(self, rows):
+        return ((float(r[0]),) for r in rows)
+    def test(self, data):
+        pho_min = data.phaseoffset.min()
+        pho_max = data.phaseoffset.max()
+        if max(abs(pho_min), abs(pho_max)) < 40:
+            return True
+        return False
+    def explain(self, data):
+        pho_min = data.phaseoffset.min()
+        pho_max = data.phaseoffset.max()
+        return {
+            'units': 'ns',
```

Comment:
hard coded units make me nervous here as it assumes something is processing the raw values. 

It should either be assumed e.g. what ever units you give us is what you get. 

Or provided with the data. In astropy there was a quantity which was a value with a unit which you could query - it actually had/has a really nice unit system.

Reply:
Not that I can see the units changing in the short but it could be attribute of the object with default value 'ns'. It will be responsabiltiy of the test calling the analyzer to change the default behavior with some knowledge of the data units:

```
def __init__(self, units='ns'):
        self._units = units
```

Reply:
See proposal https://github.com/redhat-partner-solutions/vse-sync-pp/commit/f2e4bca405af360feba1e4e2994616ecb03a5791